### PR TITLE
continous integration talk

### PR DESCRIPTION
**Pull request by chaudhary27**
- continuous integration talk by faisal

### CODE REVIEW

File: _posts/2015-09-20-Continuous-Integration.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+## Background on Faisal
+Faisal is currently attending the Firehose Project coding bootcamp and learned about continuous integration during the capstone project.
+
+Say Hello to Faisal [here](http://www.twitter.com/ffarooq00).
+
+{% include youtubePlayer.html id="905fwucM-Kc" %}
+
+Watch the full video [here](https://www.youtube.com/watch?v=905fwucM-Kc).
```
Comment (colinrubbert) [inline _posts/2015-09-20-Continuous-Integration.markdown:17]: This should be a link to the full lightning talk. 

`{% include youtubePlayer.html id="rQ8iLTFZyz0" %}`

**Comment by colinrubbert** [comment]
Just change the link and resubmit. Other than that it looks great.

**Comment by chaudhary27** [comment]
updated the link. thanks Colin :smile:

**Comment by colinrubbert** [comment]
Great job, thanks @chaudhary27

**Pull request merged**